ci/gucs.out

```diff
@@ -0,0 +1,134 @@
+"citus_tests_label_provider",
```

Comment:
These files seem to be added accidentally. Would be good to add them to `.gitignore` in another PR.

Reply:
Actually I added those by mistake :)